engine/src/execution_graph/logic_controllers/BatchOrderByProcessing.h

```diff
@@ -55,12 +55,11 @@ class SortAndSampleSingleNodeKernel : public kernel {
 											"substep"_a=context->getQuerySubstep(),
 											"info"_a="In SortAndSampleSingleNode kernel batch {} for {}. What: {}"_format(batch_count, expression, e.what()),
 											"duration"_a="");
-				logger->flush();
```

Comment:
porque estas quitando este flush?

Reply:
I'm calling `spdlog::flush_on(spdlog::level::err);` in `initialize.cpp` which does exactly the same